src/App.tsx

```diff
@@ -14,190 +53,28 @@ export const App: React.FC = () => {
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this button should have `active` class only if all todos are completed */}
-          <button
-            type="button"
-            className="todoapp__toggle-all active"
-            data-cy="ToggleAllButton"
+        <Header />
+        {todos.length > 0 && (
```

Comment:
```suggestion
        {!!todos.length && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**